### PR TITLE
More 0.33 polish

### PR DESCRIPTION
**Pull request by mazameli**
Replaces the `Get Answer` button with a play button:
image

Give join strategy button a tooltip and clean up column picker button:
image

Fix spacing for collapsed filters and lineage in titles:
image

### CODE REVIEW

File: frontend/src/metabase/query_builder/components/QueryVisualization.jsx [hunk 1]
```diff
@@ -182,8 +183,11 @@ export const VisualizationDirtyState = ({
     <RunButtonWithTooltip
       className="shadowed"
       circular
+      compact
+      style={{ width: 90 }}
       result={result}
       isRunnable={isRunnable}
+      isNative={isNative}
```
Comment (kdoh) [inline frontend/src/metabase/query_builder/components/QueryVisualization.jsx]: What does the addition of `isNative` do here?

**Review by kdoh — CHANGES_REQUESTED** [review]
A couple quick things I noticed.

The wide size of the button feels a bit awkward to me. I figure if we're going with the play style we might as well increase the click target a bit.
Screen Shot 2019-08-07 at 3 37 49 PM

Going full circle could be one option (quickly mocked below in browser), or we should trim the width down and increase the height a bit if we don't like the circle. 
Screen Shot 2019-08-07 at 3 38 43 PM

File: frontend/src/metabase/query_builder/components/QueryVisualization.jsx (hunk 1 above)
Comment (mazameli) [reply frontend/src/metabase/query_builder/components/QueryVisualization.jsx]: I'd intended to pass this prop down to RunButtonWithTooltip to let it conditionally show a tooltip on the run button to let folks know what the shortcut key is for running a query, but I didn't finish that implementation, so I'll remove `isNative` in these spots.

**Comment by mazameli** [comment]
@kdoh Thanks for the notes. Pushed a commit that fixes the spots you called out, plus updates the run button to look like this, with a new `play` icon path:
image

**Review by kdoh — APPROVED** [review]

**Pull request merged**